### DISCUSSION TITLE
Boostrap Js

### DISCUSSION THREAD

**Discussion by bsastregx** [holger1411/11straps#140] [Q&A]
Hello, thank you for providing this starter.
I have a question. Why have you copied boostrap js files to `./src/js` ?
Why don't you import directly from node_modules ?

**Comment by holger1411** [comment]
Hi!
I had several breaking changes and problems in the past pulling in files directly from the node_modules folder after an "npm update". So having a clean and working copy of all assets  in a separated folder feels just better ;-)
The command "gulp copy-assets" moves all needed files automatically from the node_modules. That commands runs automatically after "npm i" but not after "npm update". If there are various updates I can test them step by step manually
